packages/core/src/components/ModalNew/ModalFooter/ModalFooterBase/ModalFooterBase.module.scss

```diff
@@ -0,0 +1,7 @@
+.footer {
+  position: relative;
+  width: 100%;
+  padding: 20px var(--spacing-large);
```

Comment:
20 intentional?

Reply:
yep
I'll guess that when design decides on the new spacing system it would be part of it
![image](https://github.com/user-attachments/assets/4afa399d-510f-4271-9e75-ae9de46b6f31)